roles/keystone/tasks/keystone-idp.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: setup keystone idp metadata
+  shell: keystone-manage saml_idp_metadata > /etc/keystone/keystone_idp_metadata.xml
+  when: keystone.keystone_identity_provider.enabled|bool
+  tags: k2k-federation
```

Comment:
Is this metadata every going to change? Is there a need to run this task every single deploy?